src/Aspire.Hosting/Backchannel/BackchannelDataTypes.cs

```diff
@@ -194,7 +194,7 @@ internal class BackchannelLogEntry
     public required string CategoryName { get; set; }
 }
 
-internal struct CommandOutput
+internal class CommandOutput
```

Comment:
Was this to fix an issue or just for consistency with the other RPC types being reference types?